soroban-env-host/src/native_contract/token/contract.rs

```diff
@@ -226,7 +295,7 @@ impl TokenTrait for Token {
 
     // Metering: covered by components
     fn is_frozen(e: &Host, id: Identifier) -> Result<bool, HostError> {
```

Comment:
Maybe just `authorized` (10 chars exactly)?

Reply:
The issue I had with this is we would end up with both `authorize` and `authorized` (but not if we went with your idea to use `set_auth`.